README.md

```diff
@@ -119,24 +119,26 @@ messages, with a receive half and a send half that an Oak module can read from
 or write to respectively. Each half of a channel is identified by a **handle**,
 which is used as a parameter to the corresponding host function calls.
 
-At each invocation, the following channel halves are implicitly available to the Oak
-Node (see [/oak/common/handles.h](oak/common/handles.h)):
+A collection of pre-configured channel halves are available to the Oak Node.
```

Comment:
I don't think these channels should be pre-configured at all? It's up to the application configuration to specify port names for the Wasm node, so if that specifies a different port name, that's the one that will be available from the node, right?

Reply:
I've improved the text a bit for this, but I don't want to get into fully documenting the `ApplicationConfig` stuff yet.